infrastructure/Taskfile.yml

```diff
@@ -1085,6 +1085,35 @@ tasks:
       preconditions:
       - *require_site
 
+    site:environment:sync:
+      desc: |
+        Synchonizes all file and database tables from environment
+        to another on make on environment a complete copy of the source.
+      deps: [lagoon:cli:config]
+      dir: "{{.dir_env}}"
+      vars:
+        SOURCE_PROJECT: "{{.SOURCE_PROJECT}}"
+        SOURCE_ENV: "{{.SOURCE_ENV}}"
+        TARGET_PROJECT: "{{.TARGET_PROJECT}}"
+        TARGET_ENV: "{{.TARGET_ENV}}"
+      cmds:
+        - lagoon ssh --project {{.TARGET_PROJECT}} --environment {{.TARGET_ENV}} -C "drush -y rsync @lagoon.{{.SOURCE_PROJECT}}-{{.SOURCE_ENV}}:%files @self:%files -- --omit-dir-times --no-perms --no-group --no-owner --no-times --chmod=ugo=rwX --delete; drush -y sql-sync @lagoon.{{.SOURCE_PROJECT}}-{{.SOURCE_ENV}} @self; drush cr -y"
+
+    sites:webmaster:reset-moduletests:
+      desc: Reset all webmaster moduletests. It is possible to exempt some.
+      deps: [lagoon:cli:config]
+      vars:
+        webmasters:
+          sh: yq '... comments="" | .sites | with_entries(select(.value | has("plan"))) | keys | .[]' {{.dir_env}}/sites.yaml
+      cmds:
+        - for: { var: webmasters }
```

Comment:
:+1: This is very nice usage of Taskfile.

As I see it each invocation is blocking. You could consider queing them up as tasks instead as suggested in #504. This then creates the challenge of monitoring the progress of each sync 😕 .

Reply:
Then maybe we shouldn't queue them up so we can keep an eye on which fail to be queued :)